### ISSUE TITLE
Change starting position of turtle?

### ISSUE THREAD

**Issue by MarleenGilsing** [Felienne/hedy#2730]
![image](https://user-images.githubusercontent.com/80678586/168055690-f79930df-2fb8-4b5e-81b9-1b9c6096ded0.png)

This is what happens when the turtle is told to move 200 steps forward. Wouldn't it make more sense if the turtle started at the bottom of the screen (centre) and forward would mean upwards (like a car on a map).

**Comment by TiBiBa** [comment]
Good point! This should be an easy fix as we code the turtle to start at the center facing right, will take a look at this for a suitable solution! We should however keep in mind that a lot of saved turtle programs will have a completely other outcome when making this change.

**Comment by Felienne** [comment]
> (like a car on a map)

Ah I like your thinking Marleen! We have some starter code now to move the turtle in the header I think, we can easily change that!

**Comment by TiBiBa** [comment]
This is a bit harder than I thought as we are unable to read the window size at the point of initializing the turtle. But, will try to work around this to get it working.
